src/components/login.tsx

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
-import xively from '../lib/xively';
```

Comment:
The login page no longer fits in the screen (you have to scroll) in iOS, see image.
<img width="512" alt="screen shot 2018-02-28 at 11 34 11 am" src="https://user-images.githubusercontent.com/7120180/36799511-5c8d284e-1c7b-11e8-806e-af038e6db50f.png">